synapse/handlers/room_list.py

```diff
@@ -186,6 +186,10 @@ def get_order_for_room(room_id):
         logger.info("After sorting and filtering, %i rooms remain",
                     len(rooms_to_scan))
 
+        #bail if no rooms to work on
```

Comment:
PEP8 coding style mandates a space after the # - which is why the travis builds are failing